### PR TITLE
Refactor admin pages to use admin_html_head.php

### PR DESCRIPTION
**Pull request by Zen4All**
Converting files to use the new 'html_admin_head.php'

### CODE REVIEW

**Comment by Zen4All** [comment]
for ref #3634

File: admin/salemaker.php [hunk 1]
```diff
@@ -259,10 +238,10 @@ function SetCategories() {
     </head>
 <?php } ?>
 <?php if ($action == 'new' || $action == 'edit') { ?>
-       <body onload="SetCategories(); SetFocus(); init()">
+       <body onload="SetCategories();">
        <div id="spiffycalendar" class="text"></div>
 <?php } else { ?>
-       <body onload="SetFocus(); init()">
+       <body>
```
Comment (drbyte) [inline admin/salemaker.php:244]: Removing `SetFocus()` here means line 307 should have `autofocus` added to the input field attributes, shouldn't it?

**Comment by drbyte** [comment]
Typo in new filename: `admin/includes/css/attrobutes_controller.css`

File: admin/salemaker.php (hunk 1 above)
Comment (Zen4All) [reply admin/salemaker.php:244]: I'll check

**Comment by drbyte** [comment]
@Zen4All note that the admin login and password-forgotten screens look differently now, including secondary buttons of wrong colour ...

Screen Shot 2020-07-01 at 7 58 14 PM
Screen Shot 2020-07-01 at 7 58 09 PM

**Comment by Zen4All** [comment]
probably because of stylesheets being loaded in different order. I'll fix it

**Comment by drbyte** [comment]
Thanks!

**Comment by scottcwilson** [comment]
Idea for when either of you have time: a brief style guide in the documentation area, explaining the decisions that were made (significance of button color, for example) to help people keep things consistent.

**Comment by Zen4All** [comment]
> Idea for when either of you have time: a brief style guide in the documentation area, explaining the decisions that were made (significance of button color, for example) to help people keep things consistent.

Documenting is not my best side, but I'll give it a go in the coming days. Where do you want me to post it?

**Comment by scottcwilson** [comment]
PR to https://github.com/zencart/documentation
Put it in /content/dev/code

**Comment by scottcwilson** [comment]
And don't even worry about wordsmithing and formatting if that's your fear - just dump the facts in and I'll handle the rest.  Appreciated!

**Comment by drbyte** [comment]
@Zen4All since this have you run into the keepalive script showing up on the Login screen?

**Pull request merged**